### ISSUE TITLE
Sidekiq Job to Schedule Banner Updates Every 10 Minutes (Story 5)

### ISSUE THREAD

**Issue by Agile6MSkinner** [department-of-veterans-affairs/va.gov-cms#19452]
## Description
Develop a production Sidekiq job to run the banner data update script every 10 minutes, keeping the data in the pSQL database up to date. Datadog monitoring will alert Slack when the cron job fails, ensuring timely maintenance and minimizing downtime.

- Timing in Cron, YAML, etc @dsasser @SnowboardTechie please fill in helpful details
- Path for location parameter: 

**Slack Channel for alert:** #facilities-monitoring

**Sequence Note:** This task must be completed after Story 4 and can be done in parallel with Story 6.

## Testing & QA
- [ ] Ensure schedule jobs continue to function
- [ ] Code review

### Scope / Impact analysis
- Schedule Jobs

### Roles / assignments
* **Functional testing:** 
* **Code review:** 

## Acceptance criteria
- [ ] Location parameter is passed
- [ ] Sidekiq job runs the banner data update script every 10 minutes in production.
- [ ] Monitoring alerts on job failure
- [ ] Job updates banner data consistently in the database.
- [ ] Datadog Monitoring acts as Regression test plan

**Comment by jilladams** [comment]
Bryan likely to pick this up after Story 4, may be less complex than 5 points, but might be at risk for sprint just due to the work that comes before it.

**Comment by jilladams** [comment]
Bryan has started this, wrapped into Story 4-ish.

**Comment by SnowboardTechie** [comment]
This is now merged into the feature branch on #19451 and can be confirmed once that is merged as well.
